src/main/java/com/puppycrawl/tools/checkstyle/site/XdocsTemplateSink.java

```diff
@@ -37,9 +38,15 @@
  */
 public class XdocsTemplateSink extends XdocSink {
```

Comment:
@romani @nrmancuso 

https://github.com/checkstyle/checkstyle/pull/13696#discussion_r1328113588
> this class is just to have our custom xdoc template files. It is planned to be removed as part of the https://github.com/checkstyle/checkstyle/issues/13426 at which time we will be using pure doxia classes. When we remove the custom code, we will also be removing the override.
>
> I recommend we create an issue for doxia, especially if we see a way they can allow an override.

Have we done this?
Is my message clear that this is no way a permanent fix and we will have to reopen #13770 once #13426 is completed?

Reply:
I do not see easy way for now to do https://github.com/checkstyle/checkstyle/issues/13426, but may be as we complete xdoc project it will be more clear, for now we move forward as we will never move it out.

Reply:
What is not clear of https://github.com/checkstyle/checkstyle/issues/13426 ?

We just have to remove our override classes, our template file, and going back to using pure XDoc like we were before this project started? There is nothing special we are doing with the XDoc besides the template mess as project lead wanted to see changes modules were doing.

Reply:
@rnveach. I understand the structural concerns of #13426. 
I would suggest a sub phase within the proposed **mvn site phase**, that explicitly replaces all the CRLFs with LFs in the xdoc xml files. May be then we can avoid an engagement with official **doxia** project.

@romani @stoyanK7 - I would also like to know your views on my proposal?

Reply:
Why are we avoiding an engagement with doxia? That's like telling our users to fix problems with checkstyle in their own projects.

Reply:
Certainly there is a point!